README.md

```diff
@@ -63,6 +63,14 @@ There is nothing special to configure on PyFileMaker. Just make sure the
 PyFileMaker directory, which contains the file FMPro.py/FMServer.py file is
 somewhere in your Python path so that Python knows where to find the module.
 
+You can install v2.6 from pypi using pip
```

Comment:
This is old version and does not work with the new FileMaker, also the methods described in the documentation in this project (like executing scripts or parsing response to JSON format) doesn't exist in the old library. I think we shouldn't encourage people to install the old version as it's really out of date.